databricks-default-cluster-policies/variables.tf

```diff
@@ -34,3 +34,8 @@ variable "policy_map" {
   description = "Map of policy names to groups"
   type = list(map(list(string)))
 }
+
+variable "power_user_group_name" {
```

Comment:
Do you feel a default could be used to keep the naming somewhat standard over all workspaces as a default, or is it okay for workspaces to explicitly create their own name?

Reply:
yup I can put a default value in.